### PR TITLE
Typo and Grammar Improvement

### PR DESCRIPTION
**Pull request by tmsagarofficial**
This PR is to fix typos and some minor grammatical improvements.

### CODE REVIEW

**Comment by DamnClin** [comment]
Thanks a lot for your contribution but I have some doubt on some of them and I'd like to let a native english check this :)

File: documentation/hexagonal-architecture.md [hunk 1]
```diff
@@ -23,52 +23,52 @@ Here are some of the properties we are looking for when using this kind of archi
 
 "The Only Constant in Life Is Change."- Heraclitus. This can't be more true in Software (hence [the name](https://en.wikipedia.org/wiki/Software)) and, if you have just a few weeks of experience, you know that "the users don't know what they want!!!". If the first quote is true the second one is fallacious!
 
-It's not that the users don't know what they want, in fact, nobody knows. Building a Software is not done with somebody telling the others what to do, it's done with [productive partnerships](https://manifesto.softwarecraftsmanship.org/). Yes, that means people will change their mind times and times again (and this is totally fine!).
+It's not that the users don't know what they want, in fact, nobody knows. Building Software is not done with somebody telling the others what to do, it's done with [productive partnerships](https://manifesto.softwarecraftsmanship.org/). Yes, that means people will change their minds times and times again (and this is totally fine!).
```
Comment (SudharakaP) [inline documentation/hexagonal-architecture.md:26]: I haven't seen the phrase, "times and times again"; I believe this should be, 

`Yes, that means people will change their mind time and time again (and this is totally fine!)`

**Review by SudharakaP — COMMENTED** [review]
Thanks for the contribution; seems good; but double check the comment below.

**Comment by pascalgrimaud** [comment]
@tmsagarofficial : can you run prettier plz?

**Comment by pascalgrimaud** [comment]
closing as no answer, so it can be stay like this

**Comment by tmsagarofficial** [comment]
@pascalgrimaud Can I open a new issue / PR to address this specific issue. I apologize for not being able to give time to this at that point.

**Comment by pascalgrimaud** [comment]
yes, sure, no worry :)
I reopen this one, you can choose to update this PR with rebase against main branch, or open a new issue or PR, as you want

**Pull request closed without merging**